Library/Homebrew/cmd/tests.rb

```diff
@@ -3,11 +3,21 @@
 #:    Run Homebrew's unit and integration tests.
 
 require "fileutils"
+require "official_taps"
 
 module Homebrew
   def tests
+    ENV["HOMEBREW_NO_ANALYTICS_THIS_RUN"] = "1"
+
+    if ARGV.include? "--official-cmd-taps"
```

Comment:
Would you mind adding this option to the documentation comment at the top of the file?